src/index.html

```diff
@@ -15,74 +15,69 @@
     <img src="https://festivaltips.dk/wp-content/uploads/2019/05/Festival-720x340.jpg" width="100%">
 
     <img src="https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F556870729%2F1630479013993%2F1%2Foriginal.20230717-224003?w=512&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C410%2C828%2C414&s=0c927410f4bf96764e5eeecc9ddb71f4" width="100%">
-    <svg class="checkers">
+    <svg viewBox="0 0 100 100" class="checkers">
       <use xlink:href="sprite.svg#checkers"></use>
     </svg>  
-    <svg class="circle-waveform-lines">
+    <svg viewBox="0 0 100 100" class="circle-waveform-lines">
       <use  xlink:href="sprite.svg#circle-waveform-lines"></use>
     </svg>
-    <svg class="dial">
+    <svg viewBox="0 0 100 100" class="dial">
       <use xlink:href="sprite.svg#dial"></use>
     </svg>  
-    <svg class="headset">
+    <svg viewBox="0 0 100 100" class="headset">
       <use  xlink:href="sprite.svg#headset"></use>
     </svg>
-    <svg class="instagram">
+    <svg viewBox="0 0 100 100" class="instagram">
       <use xlink:href="sprite.svg#instagram"></use>
     </svg>  
-    <svg class="menu-burger">
+    <svg viewBox="0 0 100 100" class="menu-burger">
       <use  xlink:href="sprite.svg#menu-burger"></use>
     </svg>
-    <svg class="mic">
+    <svg viewBox="0 0 100 100" class="mic">
       <use xlink:href="sprite.svg#mic"></use>
     </svg>  
-    <svg class="microphne-alt">
+    <svg viewBox="0 0 100 100" class="microphne-alt">
```

Comment:
- [ ] Great effort working on the capstone project. but currently i notice you haven't followed the project requirement as much as possible. the requirement for this milestone is to have an HTML/CSS and JavaScript file that holds all your code. the HTML file should be the one that will hold the structure of the webpage, the CSS should hold the styling while the JavaScript file will hold your mobile menu functionality and creating the feature speaker dynamically. I realized the way you have structured this project is far beyond the necessary requirement and i would kindly advice you stick and maintain the requirement for this milestone.
Here is a link to the project requirement again: [LINK](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)

Also here is a link to a project example of what you are required to do: [LINK](https://github.com/mutonic/Capstone_conferencePage/tree/header). please only use this as a guide and don't do a copy and paste of the project inorder not to be penalized for plagiarism

- [ ] Also the live demo link is not working and throwing error 404. I would kindly suggest you work on the project properly and get it deployed correctly.